openvariant/variant/variant.py

```diff
@@ -37,6 +37,10 @@ def _open_file(file_path: str, mode='r+b'):
 
     return mm, file
 
+def _detect_delimiter(line: str):
+    sniffer = csv.Sniffer()
+    dialect = sniffer.sniff(line, delimiters='\t,;')
+    return dialect.delimiter
```

Comment:
I have attempted to run this method, and found out that it fails with TSV files with a single column. For these cases, the sniff function raises a `_csv.Error: Could not determine delimiter`. Because of this, we could catch this specific error, and return a fallback delimiter (`\t`) so that the script can continue.

```suggestion
def _detect_delimiter(line: str):
    sniffer = csv.Sniffer()
    try:
        dialect = sniffer.sniff(line, delimiters='\t,;')
        return dialect.delimiter
    except csv.Error as e:
        if "Could not determine delimiter" in str(e):
            return '\t'
        else:
            raise e
```